src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,40 @@
+import React = require('react');
+
+export interface CheckBoxProps {
+    text?: string;
+    value?: boolean;
+    boxIcon?: string;
+    tickIcon?: string;
+    onChange?: (value: boolean) => any;
+}
+
+export const CheckBox: React.SFC<CheckBoxProps> = (props) => (
+    <div data-automation-id="CHECKBOX_ROOT"
+         onClick={(event) => executeClickHandler(props.onChange!, !props.value)}>
+        <img data-automation-id="CHECKBOX_BOX"
```

Comment:
Hm... the intention was that boxIcon and tickIcon are React component classes (classic or stateless) so they are bundled together with the code and are not loaded afterwards. You can have a look in our other projects, we wrote stateless components returning SVG markup. Also, I am not sure how much you can style svg in img element.